atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockTables.java

```diff
@@ -53,13 +66,34 @@ public SchemaMutationLockTables(CassandraClientPool clientPool, CassandraKeyValu
                 .collect(Collectors.toSet());
     }
 
+    public boolean isLockTableEmpty(TableReference tableRef) {
+        ColumnPath columnPath = new ColumnPath(tableRef.getQualifiedName());
+        columnPath.setColumn(SchemaMutationLock.getGlobalDdlLockColumnName());
+        try {
+            clientPool.runWithRetry(client -> client.get(
+                    SchemaMutationLock.getGlobalDdlLockRowName(), columnPath, ConsistencyLevel.LOCAL_QUORUM));
+        } catch (NotFoundException e) {
+            return true;
+        } catch (TException e) {
```

Comment:
Let's log this exception. (Possibly at debug, if it's expected to happen a bunch while Cassandra is down)